dace/codegen/targets/target.py

```diff
@@ -49,7 +49,7 @@ def generate_state(self, sdfg, state, function_stream, callsite_stream):
                                     to the current location (call-site)
                                     in the code.
         """
-        raise NotImplementedError('Abstract class')
+        pass
```

Comment:
The except/pass monster shows up again  :laughing:

Reply:
This changed to pass since not all code generators implement or must implement this method :)